src/pages/components/pane.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Pane
+description: >
+  Panes give users a quick way to resize layouts vertically or horizontally by
+  interacting with the splitter. They can also be used to create expandable or
+  collapsible layouts.
+package: '@zendeskgarden/react-grid'
+components:
+  - grid/src/elements/pane/PaneProvider.tsx
+  - grid/src/elements/pane/Pane.tsx
+subcomponents:
+  - grid/src/elements/pane/components/Content.tsx
+  - grid/src/elements/pane/components/Splitter.tsx
+  - grid/src/elements/pane/components/SplitterButton.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To make a structured layout resizable, expandable or collapsible</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Vertical
+
+Panes can be used in a vertical orientation. The panes resize up to the values
+that you set.
+
+import PaneVertical from '../../examples/pane/PaneVertical.tsx';
+import PaneVerticalCode from '!!raw-loader!../../examples/pane/PaneVertical.tsx';
+
+<CodeExample code={PaneVerticalCode}>
+  <PaneVertical />
+</CodeExample>
+
+### Horizontal
+
+Panes can be used in a horizontal orientation as well.
+
+import PaneHorizontal from '../../examples/pane/PaneHorizontal.tsx';
+import PaneHorizontalCode from '!!raw-loader!../../examples/pane/PaneHorizontal.tsx';
+
+<CodeExample code={PaneHorizontalCode}>
+  <PaneHorizontal />
+</CodeExample>
+
+### Grid
+
+Panes can be resized in both vertical and horizontal directions for a complex
+grid flexible layout.
+
+import PaneGrid from '../../examples/pane/PaneGrid.tsx';
+import PaneGridCode from '!!raw-loader!../../examples/pane/PaneGrid.tsx';
+
+<CodeExample code={PaneGridCode}>
+  <PaneGrid />
+</CodeExample>
+
+## Customization
+
+Panes can be collapsed or expanded by using the [SplitterButton](#panesplitterbutton).
+
+### Vertical collapse
+
+import PaneVerticalCollapse from '../../examples/pane/PaneVerticalCollapse.tsx';
+import PaneVerticalCollapseCode from '!!raw-loader!../../examples/pane/PaneVerticalCollapse.tsx';
+
+<CodeExample code={PaneVerticalCollapseCode}>
+  <PaneVerticalCollapse />
+</CodeExample>
+
+### Horizontal collapse
+
+import PaneHorizontalCollapse from '../../examples/pane/PaneHorizontalCollapse.tsx';
+import PaneHorizontalCollapseCode from '!!raw-loader!../../examples/pane/PaneHorizontalCollapse.tsx';
+
+<CodeExample code={PaneHorizontalCollapseCode}>
+  <PaneHorizontalCollapse />
+</CodeExample>
+
+### Splitter button position
```

Comment:
Changed from "Icon button position" to "Splitter button position" for consistency. Let me know if this works @lucijanblagonic and @diana-oum .

Reply:
👍 @smritimadan